packages/@aws-cdk/app-delivery/lib/pipeline-deploy-stack-action.ts

```diff
@@ -69,16 +98,23 @@ export class PipelineDeployStackAction extends cdk.Construct {
       throw new Error(`createChangeSetRunOrder (${createChangeSetRunOrder}) must be < executeChangeSetRunOrder (${executeChangeSetRunOrder})`);
     }
 
-    this.stack = props.stack;
     const changeSetName = props.changeSetName || 'CDK-CodePipeline-ChangeSet';
 
-    new cfn.PipelineCreateReplaceChangeSetAction(this, 'ChangeSet', {
+    this.stack = props.stack;
+
+    const fullPermissions = props.fullPermissions === true;
```

Comment:
I don't think this is needed. `fullPermissions` defaults to `false` in `PipelineCreateReplaceChangeSetAction` (and I reckon we're quite unlikely to change this default, as it has pretty drastic security implications).